config/routes.rb

```diff
@@ -16,8 +16,6 @@
   devise_for :users
   Hydra::BatchEdit.add_routes(self)
 
-  curation_concerns_collections
```

Comment:
Not Sure. They may have gotten lost when merging from master. I can reinstate them.